blocks/cta/cta.css

```diff
@@ -36,8 +36,7 @@
 
 /* CTA Button */
 .section.cta-container .cta.block .button-container {
-    margin: 0 auto;
-    padding-top: 32px;
+    margin: var(--spacer-element-08) auto 0 auto;
```

Comment:
CTA buttons are not able to inherit global button styling?

Reply:
Not sure, I didn't work on buttons, just the margin of the P needed to be reduced here.

Reply:
I think this should be applied to ALL button containers. I would prefer the CTA block not have its own special version of a button since I am pretty sure buttons are the same across all Figma designs.

Reply:
I think it is the spacing that may be unique in the CTA space. I don't know where all the uses of the button are, so I'm not comfortable with adding the special margin to all at this time.